header.php

```diff
@@ -25,7 +25,11 @@
 	flush();
 	?>
 </head>
-<body <?php body_class( 'js-enabled' ); ?>>
+<body <?php body_class(""); ?>>
+<script>
```

Comment:
What was the thinking behind putting it in a script tag vs in the JS file?

Reply:
This the the approach taken by GDS - I have not investigated why they choose this method.  However, you can see it is the way they tell people to do it [in their get started guide](https://frontend.design-system.service.gov.uk/get-started/#5-get-the-javascript-working).